python/sglang/srt/layers/attention/utils.py

```diff
@@ -30,20 +101,20 @@ def create_flashinfer_kv_indices_triton(
         kv_start = tl.load(kv_start_idx + pid).to(tl.int32)
         kv_end = kv_start
     kv_end += tl.load(page_kernel_lens_ptr + pid).to(tl.int32)
-
-    num_loop = tl.cdiv(kv_end - kv_start, BLOCK_SIZE)
-    for i in range(num_loop):
-        # index into req_to_token_ptr needs to be int64
-        offset = tl.arange(0, BLOCK_SIZE).to(tl.int64) + i * BLOCK_SIZE
-        mask = offset < kv_end - kv_start
-        data = tl.load(
-            req_to_token_ptr
-            + req_pool_index * req_to_token_ptr_stride
-            + kv_start
-            + offset,
-            mask=mask,
-        )
-        tl.store(kv_indices_ptr + kv_indices_offset + offset, data, mask=mask)
+    
+    kv_range = kv_end - kv_start
+    num_pages = tl.cdiv(kv_range, PAGE_SIZE)
+    num_pages_loop = tl.cdiv(kv_range, BLOCK_SIZE)
+    base_ptr = req_to_token_ptr + req_pool_index * req_to_token_ptr_stride + kv_start 
+    for i in range(num_pages_loop):
```

Comment:
nit: I feel this can be still called num_loop to avoid confusion, since the block_size is just kernel level config.